lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1313,6 +1302,7 @@ def cluster_name_from_kube_env(kube_env)
     end
 
     def compute_timestamp(resource_type, record, time)
+      current_time = Time.now
```

Comment:
[Optional] `now = Time.now`.

Reply:
Ah, in fact I was trying to avoid having the variable name the same as a method name.

Reply:
Why? The method is in a different class, and you're storing its return value in the variable...

Reply:
Because... it feels like we are calling the method again?

Reply:
Ok, makes sense.